subsys/usb/class/cdc_acm.c

```diff
@@ -301,6 +301,18 @@ static void tx_work_handler(struct k_work *work)
 	len = ring_buf_get_claim(dev_data->tx_ringbuf, &data,
 				 CONFIG_USB_CDC_ACM_RINGBUF_SIZE);
 
+	/*
+	 * Send less data to avoid having to send a zero-length packet. When
```

Comment:
I suggest to avoid word "send" here, one could believe that it is possible to send data to the host like over Ethernet or UART.

~~Send less data to avoid having to send a zero-length packet. When
send a zero-length packet, the host or t~~he application running on
the host may believe that there is no more data to be received
(i.e. the transaction has completed), hence not triggering another
~~IN transaction, and thus preventing any more data to be sent from
Zephyr to the Host.~~

Transfer less data to avoid zero-length packet. The application running on
the host may conclude that there is no more data to be received
(i.e. the transaction has completed), hence not triggering another
I/O Request Packet (IRP).